src/components/SplashScreen/SplashScreen.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { render } from '@testing-library/react'
+import { SplashScreen } from '.'
+
+describe('SplashScreen component', () => {
+  it('renders the splash screen', () => {
+    render(<SplashScreen />)
+    const splashScreen = document.querySelector(
+      'div.flex.flex-col.justify-center.h-full.w-full'
```

Comment:
Instead of querying class names, I would suggest to go the more React Testing Library way and mimic the user as much as possible. In that sense I would consider the test passing when I see the actual content rendered, e.g. the headline, buttons, or the logo. Perhaps you can make some use of `getByRole` and `getByAltText` here?